static/src/javascripts/lib/cookies.spec.js

```diff
@@ -0,0 +1,116 @@
+// @flow
+import {
+    cleanUp,
+    add,
+    addSessionCookie,
+    addForMinutes,
+    remove,
+    get,
+} from 'lib/cookies';
+
+jest.mock('lodash/arrays/remove', () => jest.fn());
+
+// Mock the Date constructor to always return the beginning of time
+const OriginalDate = global.Date;
+global.Date = jest.fn(() => new OriginalDate(0));
```

Comment:
More as a question, but why do we override the global Date here, and what are the options to avoid this? Although, I do like that we've got a check against not fixing this at the bottom.